Nodejs/Product/Nodejs/Project/NodejsProjectNode.cs

```diff
@@ -95,6 +96,33 @@ public NodejsProjectNode(NodejsProjectPackage package)
                     fileNode.Analyze();
                 }
             }
+
+            TryToAcquireTypings();
+        }
+
+        private bool ShouldAcquireTypingsAutomatically {
+            get {
+                return !_projectHasTypeScriptFiles
+                    && NodejsPackage.Instance.IntellisenseOptionsPage.EnableES6Preview
+                    && NodejsPackage.Instance.NpmOptionsPage.EnableAutomaticTypingsAcquisition;
```

Comment:
Dev15 Salsa is supposed to automatically acquire typings, so let's ifdef this check so that the feature only lights up in Dev14.